paper-slider.html

```diff
@@ -292,7 +292,7 @@
      * @method increment
      */
     increment: function() {
-      this.value = this._clampValue(this.value + this.step);
+      this.value = this._clampValue(this.value + this.step).toString();
```

Comment:
@jklein24 @cdata this is gonna be problematic, because now we will be concatenating strings in ```this.value + this.step``    e.g.  if value is `0.1` and step `0.1`,  ` _clampValue`  that expression will be equal to `0.10.1`